iron-component-page.html

```diff
@@ -480,6 +478,11 @@
           } else if (this.src) {
             srcUrl = new URL(this.src, this.base).toString();
           } else {
+            // Remove deep link hash from base URL.
+            var url = new URL(this.base);
+            url.hash = '';
+            this.base = url.toString();
```

Comment:
Alternatively, you could remove the hash from the default value of `base` which is `this.ownerDocument.baseURI`.
e.g.
` return this.ownerDocument.baseURI.replace(/\#.*$/,'');`